plugins/modules/foreman_compute_resource.py

```diff
@@ -154,20 +154,20 @@
 from ansible.module_utils.foreman_helper import ForemanEntityApypieAnsibleModule
 
 
-def get_provider_infos(provider):
+def get_provider_param_keys(provider):
     provider_name = provider.lower()
```

Comment:
is this really required btw?

Reply:
I don't know, but it allows for some validation before the server api yells at us.